ivy/functional/frontends/torch/nn/functional/convolution_functions.py

```diff
@@ -57,7 +55,18 @@ def _conv_transpose(
     padding, output_padding = map(
         lambda x: [x] * dims if isinstance(x, int) else x, [padding, output_padding]
     )
-    pad_widths = [(weight.shape[i] - 1,) * 2 for i in range(dims)]
+    dilation = [dilation] * dims if isinstance(dilation, int) else dilation
+    stride = [stride] * dims if isinstance(stride, int) else stride
+
```

Comment:
Also, could you please remove the dilation and padding limitations of the corresponding tests? 
You would need to remove the condition [here](https://github.com/unifyai/ivy/blob/d3c562dcc91cd839e3e1a05bee0deb2fbda7c02e/ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_convolution_functions.py#L232) and the use of `max_dilation` [here](https://github.com/unifyai/ivy/blob/d3c562dcc91cd839e3e1a05bee0deb2fbda7c02e/ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_convolution_functions.py#L367) and [here](https://github.com/unifyai/ivy/blob/d3c562dcc91cd839e3e1a05bee0deb2fbda7c02e/ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_convolution_functions.py#L447). Hope I'm not forgetting something.